### ISSUE TITLE
TS: Can not pass a number to a query()

### ISSUE THREAD

**Issue by ecdeveloper** [danpaz/bodybuilder#178]
After [this](https://github.com/danpaz/bodybuilder/commit/79ebbc05da155f4f85079b8cd13733ee79913b38#diff-552f7ba88730ac230cfd8b419068f3e5R7) recent change, I'm getting a TS error when doing 

```javascript
bodybuilder().query("match", "someId", someId)
```

where `someId` is a number. 

I think it's not huge of a deal, I can change my typing to have it as string (even though its stored as an `integer` in elastic). But I'm just curious why numbers weren't considered to be added to typings.

**Comment by ferronrsmith** [comment]
The definitions doesn't really affect the library itself. 

```javascript
bodybuilder().query("match", "someId", 2).build()
```

produces (for me) : 

```json
{
  "query": {
    "match": {
      "someId": 2
    }
  }
}
```

**Comment by danpaz** [comment]
I don't use TS but my guess is it fails to compile because our type defs
enforce string instead of number

**Comment by ferronrsmith** [comment]
Working on adding tests to the definitions

**Comment by ecdeveloper** [comment]
Yeah, @danpaz, you are right. I just can not compile my TS scripts, since the interface requires either an object or a string.

**Comment by ferronrsmith** [comment]
@ecdeveloper can you test against the #chore/typings branch and see if everything works. I have added most of what I think could be queried, but I am sure there are a few complex queries I am missing.

**Comment by ecdeveloper** [comment]
Yup! It seems to be working fine now.

**Comment by danpaz** [comment]
Closed by #179, please install v2.2.10
